integration-tests/feature_list.txt

```diff
@@ -20,7 +20,7 @@ features/stack_analysis_pypi_direct_and_transitive.feature
 features/stack_analyses_maven_ecosystem.feature
 features/stack_analysis_maven_direct_and_transitive.feature
 features/stack_analyses_npm_ecosystem.feature
-features/stack_analysis_npm_direct_and_transitive.feature
+#features/stack_analysis_npm_direct_and_transitive.feature
```

Comment:
Exact same tests are present in 
https://github.com/sawood14012/fabric8-analytics-common/blob/master/integration-tests/features/stack_analyses_smoke_tests_npm_ecosystem.feature